src/App.js

```diff
@@ -1,15 +1,21 @@
 import React from "react";
-
+import Header from "./Heading"
```

Comment:
Its normally best practise to name the component and the file the same thing since it makes searching for them much easier.